docs/index.md

```diff
@@ -50,6 +50,7 @@ For the Scala API, Spark {{site.SPARK_VERSION}}
 uses Scala {{site.SCALA_BINARY_VERSION}}. You will need to use a compatible Scala version
 ({{site.SCALA_BINARY_VERSION}}.x).
 
+For Python 3.9, Apache Arrow's PyArrow library is not compatible. Please check [Python Compatibility](https://arrow.apache.org/docs/python/install.html#python-compatibility) page.
```

Comment:
@dongjoon-hyun, how about:

```suggestion
For Python 3.9, it will be supported in Apache Arrow 3.0+. Please check the latest [Python Compatibility](https://arrow.apache.org/docs/python/install.html#python-compatibility) page.
```

Reply:
related JIRAs:
- https://issues.apache.org/jira/browse/ARROW-10905
- https://issues.apache.org/jira/browse/ARROW-10904
- https://issues.apache.org/jira/browse/ARROW-10224

Reply:
Thanks. However, it may sounds inappropriate to claim some other open source community's suggestion. Apache Spark also drop Scala 2.13 from Spark 3.1.0.

Reply:
and seems like they are running RCs for 3.0.0 (https://mail-archives.apache.org/mod_mbox/arrow-dev/202101.mbox/browser)

Reply:
IMO, `will be supported in Apache Arrow` is not what we can say in the Apache Spark official website.

Reply:
On the other hand, it will be incorrect when Spark 3.1.1 is released as it looks Arrow 3.0.0 is likely released almost same timing.

Reply:
How about just simply say like "For Python 3.9, Arrow optimization and pandas UDFs might not work due to the supported Python versions in Apache Arrow. Please refer to the latest Python Compatibility"?

Reply:
That sounds reasonable for now.